src/Microsoft.ML.Transforms/CustomMappingTransformerFactory.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Microsoft.Data.DataView;
+
+namespace Microsoft.ML.Transforms
+{
+    /// <summary>
+    /// Place this attribute onto a type to cause it to be considered a custom mapping transformer factory.
```

Comment:
>custom mapping transformer factory [](start = 72, length = 34)

Actually, pursuant to my last comment about documentation, we'd probably want to do another thing here... `<see` tags to and from `CustomMappingEstimator`, the usage of that via `MLContext` (if appropriate), and so on, seems like a larger task that ought to be covered via documentation team. Maybe file an issue if you feel like approaching it right now is a bit much (which I could easily understand)?

Reply:
#2657 will handle this as well.